src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -291,7 +294,10 @@ def _wrap_modules(self, layer, config):
             the configuration for generating the mask
         """
         _logger.info("compressing module %s.", layer.name)
-        return PrunerModuleWrapper(layer.module, layer.name, layer.type, config, self)
+        wrapper = PrunerModuleWrapper(layer.module, layer.name, layer.type, config, self)
+        assert hasattr(layer.module, 'weight')
```

Comment:
why not using `check_weight` method?

Reply:
where is this method defined, and how to use it?

Reply:
https://github.com/microsoft/nni/blob/d452a1665f802bfbe75372b6c942272b252c70a2/src/sdk/pynni/nni/compression/torch/compressor.py#L563